pkg/adapter/adapter.go

```diff
@@ -183,9 +186,21 @@ func (a *Adapter) ConsumeMessages(channel *wabbit.Channel,
 func (a *Adapter) PollForMessages(channel *wabbit.Channel,
 	queue *wabbit.Queue, stopCh <-chan struct{}) error {
 	logger := a.logger
+	var err error
+	if a.config.BackoffDelay != "" {
+		retriesInt32 = int32(a.config.Retry)
+		backoffPolicy := utils.SetBackoffPolicy(a.context, a.config.BackoffPolicy)
```

Comment:
`utils.SetBackoffPolicy` can still return a "". Should we continue checking for that?